site/en/docs/webstore/identify_user/index.md

```diff
@@ -5,84 +5,204 @@ date: 2017-08-30
+// background.js
+
+chrome.action.onClicked.addListener(function () {
+  console.log('action clicked')
+})
+```
+
+{% include 'partials/extensions/reusing-prod-extension-id.md' %}
+
+## Get the OAuth client ID {: #get-client-id}
+
+Navigate to the [Google API Console][google-console] and create a new project. To get a OAuth client ID, follow these steps:
+
+**1. Customize the consent screen and select scope.**
+   - Click the **OAuth consent screen** menu item.
+   - Fill out the required consent screen information.
+   - Add the **openid** scope.
+   - Add **Test users**.
+   - Click **Save > Continue**.
+  
+**2. Create credentials.**   
+   - Click the **Credentials** menu item.
+   - Click **Create new credentials > Auth Client ID**.
+   - Select Application type **Web application**.
+   - Enter the **Name** of the OAuth2 client
+   - Add `https://<YOUR_EXTENSION_ID>.chromiumapp.org/` as the **Authorized redirect URI**.
+   - Finish by clicking **Create**. 
+
+The console will provide an OAuth client ID. Keep this ID for later use.
+
+## Launch the Authorization flow {: #auth-flow}
+
+### Request the "identity" permission {: #identity-permission}
+
+To use the [Chrome Identity API][identity-api], declare the `"identity"` permission in the `manifest.json`.
+
+```json
+{
+  "name": "Google OpenID Connect example",
+  ...
+  "permissions": [
+    "identity"
+  ],
+  ...
+}
+```
+
+### Construct the authorization URL {: #auth-url}
+
+Before the extension can make a request to Google’s OAuth2 endpoint using the Identity API,
+you need to construct an [authorization URL][auth-url]. It must contain several request parameters
+including the client ID and redirect URI. In addition to the `openid` [scope][openid-scopes], you can request
+`profile` information and/or `email`. Update `background.js` to match the following code:
+
+```javascript
+// background.js
+
+let clientId = '<CLIENT_ID>'
+let redirectUri = `https://${chrome.runtime.id}.chromiumapp.org/`
+let nonce = Math.random().toString(36).substring(2, 15)
+
+chrome.action.onClicked.addListener(function () {
+  const authUrl = new URL('https://accounts.google.com/o/oauth2/v2/auth')
+
+  authUrl.searchParams.set('client_id', clientId)
+  authUrl.searchParams.set('response_type', 'id_token')
+  authUrl.searchParams.set('redirect_uri', redirectUri)
+  // Add the OpenID scope. Scopes allow you to access the user’s information.
+  authUrl.searchParams.set('scope', 'openid profile email')
+  authUrl.searchParams.set('nonce', nonce)
+  // Show the consent screen after login
+  authUrl.searchParams.set('prompt', 'consent')
+
+})
+
+```
+
+Replace `<CLIENT_ID>` with the API key generated from the Google console. 
+
+### Retrieve a redirect URL {: #redirect-url}
+
+Now that the extension has the client ID, redirect URI, and OAuth URL, it can initiate Google's
+authentication flow. Call [`identity.launchWebAuthFlow()`][identity-webauthflow] to launch the web auth flow and
+retrieve a redirect URL. 
```

Comment:
```suggestion
Now that the extension has the client ID, redirect URI, and OAuth URL, it can initiate Google's
authentication flow. Call [`identity.launchWebAuthFlow()`][identity-webauthflow] to launch the web
auth flow and retrieve a redirect URL. 
```

Rewrap at 100 characters.